docs/tutorial/piezotracking.rst

```diff
@@ -142,9 +142,9 @@ distance without the tether. :math:`F_{1x}` and :math:`F_{2x}` are the forces me
 
 To do this in Pylake, we set up the piezo distance calibration as follows::
 
-    piezo_calibration = lk.PiezoForceDistance(distance_calibration, baseline_1x, baseline_2x)
+    piezo_calibration = lk.PiezoForceDistance(distance_calibration, baseline_1x, baseline_2x, signs=(1, - 1))
 
-We now have all the calibrations we need to do piezo tracking on our experimental data.
+where `signs` contains the sign of force 1x and force 2x respectively. We now have all the calibrations we need to do piezo tracking on our experimental data.
```

Comment:
```suggestion
where `signs` specifies the sign of force 1x and force 2x respectively. We now have all the calibrations we need to do piezo tracking on our experimental data.
```

Are the signs often different from the default? Should we tell people in the tutorial how to determine them?

Reply:
I updated the text and squashed the commits, let me know your thoughts.